src/components/sbb-radio-button-group/sbb-radio-button-group.tsx

```diff
@@ -0,0 +1,256 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  h,
+  Host,
+  JSX,
+  Listen,
+  Prop,
+  State,
+  Watch,
+} from '@stencil/core';
+import getDocumentWritingMode from '../../global/helpers/get-document-writing-mode';
+import {
+  createNamedSlotState,
+  queryAndObserveNamedSlotState,
+  queryNamedSlotState,
+} from '../../global/helpers/observe-named-slot-changes';
+import { InterfaceSbbRadioButton } from '../sbb-radio-button/sbb-radio-button.custom';
+import { InterfaceSbbRadioButtonGroup } from './sbb-radio-button-group.custom';
+
+let nextId = 0;
+type RadioButton = InterfaceSbbRadioButton & HTMLElement;
```

Comment:
I think you could just use `HTMLSbbRadioButtonElement`, which is generated in src/components.d.ts